Integrations/Flashpoint/README.md

```diff
@@ -0,0 +1,66 @@
+Use flashpoint integration for reduce business risk. The integration was
+integrated and tested with Demisto 5.0 This integration was integrated
+and tested with version xx of Flashpoint
```

Comment:
please generate README.md according to https://github.com/demisto/content/tree/master/docs/integration_documentation

Reply:
We generated and added the readme

Reply:
Have you generated according to the link I provided? Because HTMLDocAutomation should generate HTML and the README you provided is not HTML